browser/js/content.js

```diff
@@ -378,6 +401,43 @@ function postRedditReply(commentNode, text) {
   replyForm.dispatchEvent(new Event("submit"));
 }
 
+// Install closing button (usually the little "x" in the corner)
+function installCloser(buttons, closeTarget, skipCheck, closeCallback) {
+  for (let closer of buttons) {
+    closer.addEventListener("click", function(e) {
+      e.preventDefault();
+      if (removeChat(closeTarget, skipCheck)) {
+        closeCallback !== undefined && closeCallback();
```

Comment:
likely safer to do a falsey check , or if someone does
```
installCloser([], el, false, null)
```

it'll pass closeCallback!==undefined and dereference the null
```
likely safer to just do
if (closeCallback) {
  closeCallback() 
}
```
or 

```
closeCallback && closeCallback()
```